schemes/zhang_mcfarlane/zm_conv_evap.F90

```diff
@@ -236,7 +236,7 @@ subroutine zm_conv_evap_run(ncol, pver, pverp, &
     end do
 
 ! set output precipitation rates (m/s)
-! convert from precip to kg/m2/s
+! convert from 'kg m-2 s-1' to 'm s-1'
```

Comment:
I really think we should emphasize that the `1000._kind_phys` below is the density of fresh water, as this technically has physical implications (as opposed to just a pure unit conversion).

Reply:
good point - I put in a variable for it.